src/CoherentSolutions.Extensions.Hosting.ServiceFabric/CoherentSolutions.Extensions.Hosting.ServiceFabric.csproj

```diff
@@ -1,7 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <NetStandardImplicitPackageVersion>2.0.0</NetStandardImplicitPackageVersion>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
Could it be because multi-targeting is required (see the next change), since 3.1 requires Microsoft.AspNetCore.App instead of Microsoft.AspNetCore in 2.2?

Reply:
> Could it be because multi-targeting is required (see the next change), since 3.1 requires Microsoft.AspNetCore.App instead of Microsoft.AspNetCore in 2.2?

Thanks, for the idea! I will try it.